README.md

```diff
@@ -20,12 +20,24 @@ just check the resulting files directly into the repo under `./content/reference
 
 ### Prerequisites
 
+- [Node.js and npm](https://www.npmjs.com/get-npm)
+- [Ruby](https://www.ruby-lang.org/en/downloads/)
```

Comment:
Do we still have a dependency on Ruby?

Reply:
Not to my knowledge. Probably safe to remove.

Reply:
Unfortunately I failed to add why we had to install Ruby in my notes. @metral do you remember what necessitated the Ruby installation?

Reply:
We used to use Jekyll for the docs site, which uses Ruby. Now that we’ve moved to Hugo, I don’t think it’s necessary anymore.

Reply:
IIRC we only installed ruby, and the pkgs in [1] because running `make build` kept yelling at us after having run `make ensure`. This could be tied to our package.json not being completely updated, but i have not dived into this further - we should probably bake and clean this up if we haven't

1 - https://github.com/pulumi/docs/pull/1310/files#diff-04c6e90faac2675aa89e2176d2eec7d8R121

Reply:
Sorry just now circling back to this: I uninstalled Ruby, and `make build` still worked, yet running `make ensure` resulted in the following error:

```
make ensure
yarn install
/usr/bin/env: ‘ruby’: No such file or directory
make: *** [Makefile:19: ensure] Error 127
```